src/Components/Artwork/Details/Details.tsx

```diff
@@ -366,17 +366,17 @@ export const Details: React.FC<DetailsProps> = ({
 
     if (!saveOnlyToDefaultList) {
       return (
-        <SaveArtworkToListsButtonFragmentContainer
+        <SaveArtworkToListsButtonQueryRenderer
```

Comment:
This is the main 'type' of change, continue to prop-drill this analytics prop, but instead of passing in the masked fragment via the `artwork` prop, pass in the artwork id to the query renderer.